packages/wp-story-editor/src/components/mediaUpload/mediaPicker/WordPressImageCropper.js

```diff
@@ -53,7 +53,7 @@ const WordPressImageCropper = window.wp?.media?.controller?.Cropper?.extend?.({
         : control.params.height;
     }
 
-    return wp.ajax.post('crop-image', {
+    return window.wp.ajax.post('crop-image', {
```

Comment:
🤔 Is this required or nice to have?

Reply:
It is required. I removed `wp` as a recognized global variable in the ESLint config, so that we are forced to explicitly write the full variable with `window.wp`.